app/src/main/java/com/x8bit/bitwarden/ui/auth/feature/startregistration/StartRegistrationScreen.kt

```diff
@@ -264,18 +263,14 @@ private fun StartRegistrationContent(
                     .testTag("RegionSelectorDropdown"),
             )
             if (isNewOnboardingUiEnabled) {
-                IconButton(
+                BitwardenStandardIconButton(
+                    iconRes = R.drawable.ic_tooltip_small,
+                    contentDescription = stringResource(R.string.help_with_server_geolocations),
                     onClick = handler.onServerGeologyHelpClick,
                     // Align with design but keep accessible touch target of IconButton.
                     modifier = Modifier.offset(y = (-8f).dp, x = 16.dp),
-                ) {
-                    Icon(
-                        painter = rememberVectorPainter(id = R.drawable.ic_tooltip_small),
-                        contentDescription = stringResource(R.string.help_with_server_geolocations),
-                        tint = MaterialTheme.colorScheme.primary,
-                        modifier = Modifier.size(18.dp),
-                    )
-                }
+                    contentColor = MaterialTheme.colorScheme.primary,
```

Comment:
Don't we typically have the modifier as the last param?

Reply:
Not that I have seen

Reply:
I can only find examples where its either the first or last param for whatever that is worth.